README.md

```diff
@@ -63,6 +63,10 @@ sudo modprobe binder_linux
 
 ```bash
 docker run -td --name aind --privileged -p 5900:5900 -v /lib/modules:/lib/modules:ro aind/aind
+```
+
+```bash
+docker run -td --name aind --privileged -p 8080:8080 -e "NOVNC=1" -v /lib/modules:/lib/modules:ro aind/aind
```

Comment:
s/NOVNC/WEBMODE/, and needs explanation

Reply:
like this maybe?

Reply:
@AkihiroSuda any thoughts on the wording in the read me?